### ISSUE TITLE
Project update 2/25 (2)

### ISSUE THREAD

**Issue by racheljfu** [cngish98/Language-of-Dickinson#2]
In our project meeting for this week, my group and I discussed further planning on our project and made plans for the upcoming week. We solidified our research question (so we will focus on elision in the poems), and discussed possibly trying to use network analysis or topic modeling later on in our project, as well as reviewing some basic commands for Github (Charlie pushed a Git commands cheat sheet to our repository for us). We also discussed how we might proceed with markdown, since elision may be hard to analyze if you are trying to analyze something that is not there (we will probably start with using empty elements in our markdown). Another issue was navigating the different editions of Dickinson's poems (since there are often many differing versions), and sorting by theme or year. Nonetheless, we're sure that these problems might become easier to solve once we gather poems to work with for the upcoming week. We plan to search for some poems and then send them to the Slack channel and save them on the Dickinson archive website (which I will primarily work on), as well as make a Relax NG schema for them, and Caroline said that she will talk more with her linguistics advisor about other possible linguistic elements we might be able to analyze in the poems.

**Comment by stanickld** [comment]
I agree that the issue of navigating different editions of Dickinson's poems might be difficult to navigate at first; because there's so many, sorting by theme or year will definitely be crucial to how you guys choose to proceed with marking them up. Also, it's great that you have a linguistics advisor to help you think of linguistics elements to mark up! In terms of the Command Line, my project partner and I also have been getting comfortable with pushing and pulling things! It can be really confusing at first but once you do it a few times you get the hang of it.

**Comment by ajm324** [comment]
I think that your analysis of elision, which I had to look up but sounds very interesting, will be a unique angle that could help you come to some cool conclusions about the text. I agree that it may be conceptually difficult to think about how to mark the absence of something and I would love to see how your group ends up coping with this and how your empty elements structure functions if that is what you decide to use. Are you looking to analyze different versions of the same poem? or to choose a consistent edition (possibly one collection) from which to draw your source text? If there are differences of elision between editions of the same piece, that could be a fun pattern to use. So far, you guys sound to be at a good place!
